src/app/components/code-block/index.html

```diff
@@ -0,0 +1,54 @@
+<sky-demo-page
+  pageTitle="Code block"
+  skyuxModule="SkyCodeBlockModule"
+  packageName="lib-code-block"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The code block component formats a block of text as a code snippet with syntax highlighting. The code block also has a header that displays a copy button and, if defined, the language type. To create a code snippet, wrap <stache-code>sky-code-block</stache-code> tags around the code block.
```

Comment:
In the last sentence, I think "snippet" is the wrong word. Generally, it seems like we use "snippet" for inline text formatted as code within a block of text, and we use "block" for larger chunks of text formatted as code in a separate block of text. I don't know that this is a hard and fast rule for these two terms, but that's how we tend to use them, so I'd avoid mixing up the terms if possible. (This on the Stache docs as well.)

Reply:
How about swapping out "snippet" with "sample" instead?

Reply:
Oops corrected the wrong sentence initially so I fixed that. I swapped it out to say "To create a code block..."